imports/models/BoardFavoritesCollection.ts

```diff
@@ -0,0 +1,35 @@
+import { BaseCollection } from './BaseCollection'
+import { BoardRecord } from './BoardCollection'
+
+export interface BoardFavoriteRecord {
+  userid: string
+  boardId: string
+}
+
+export class BoardFavoritesCollection extends BaseCollection<BoardFavoriteRecord> {
+  constructor() {
+    super('board_favorites')
+  }
+
+  public async addFavoriteBoard(userid: string, boardId: string): Promise<void> {
+    this.collection.update({ userid, boardId }, { userid, boardId }, { upsert: true })
+  }
+
+  public async removeFavoriteBoard(userid: string, boardId: string): Promise<void> {
+    await this.remove({ userid, boardId })
```

Comment:
unnecessary `await` here

Reply:
maybe we can chagne these two methods `addFavoriteBoard` and `removeFavoriteBoard` 's return value to `Promise<boolean>`, and `boolean` means whether this operation is success or not.

What do you think of it?